GitForWindowsHelper/slash-commands.js

```diff
@@ -173,10 +162,43 @@ module.exports = async (context, req) => {
                     text
                 )
             if (!isMSYSPackage(package_name)) {
+                await queueCheckRun(
+                    context,
+                    await getToken(),
+                    'git-for-windows',
+                    repo,
+                    ref,
+                    'deploy',
+                    `Build and deploy ${package_name}`,
+                    `Deploying ${package_name}`
+                )
+
                 const answer = await triggerBuild()
                 const answer2 = await appendToComment(`The workflow run [was started](${answer.html_url})`)
                 return `I edited the comment: ${answer2.html_url}`
             }
+            
+            await queueCheckRun(
+                context,
+                await getToken(),
+                'git-for-windows',
+                repo,
+                ref,
+                'deploy_x86_64',
+                `Build and deploy ${package_name}`,
+                `Deploying ${package_name}`
+            )
+            await queueCheckRun(
+                context,
+                await getToken(),
+                'git-for-windows',
+                repo,
+                ref,
+                'deploy_i686',
```

Comment:
Good idea, but since we want to reuse the check run, we need a corresponding change to [git-for-windows-automation](https://github.com/git-for-windows/git-for-windows-automation/blob/a538b16263ff442be4729cf990f1bf4001e5dc6c/.github/workflows/build-and-deploy.yml#L93): instead of a simple `deploy` it needs to use something like:

```javascript
const checkRunName = `${
    process.env.BUILD_ONLY ? 'build' : 'deploy'
  }${
    process.env.ARCHITECTURE ? `_${process.env.ARCHITECTURE}` : ''
  }`
```

And I think we need that change ready to go before we merge this here Pull Request.

Reply:
Oh, I see that you already did that [here](https://github.com/git-for-windows/git-for-windows-automation/pull/7/commits/acb41aa6f99419c451ebde30e2e8909a76e16a3c#diff-ecad2183af1362d9d99f4a33e7491c1970c92255c3c318e688ece549ec91fe33R93)...